setup/docker.md

```diff
@@ -91,6 +91,45 @@ For example:
 docker run --rm -it -p 7080:7080 -v /var/run/docker.sock:/var/run/docker.sock -v ~/.coder:/var/run/coder -e DEVURL_HOST="*.mycompany.com" codercom/coder:1.27.0
 ```
 
+## External PostgreSQL database
+
+If you'd like to use an external database, you must:
+
+1. Disable the embedded database with the environment variable `DB_EMBEDDED`
+1. pass in the database connection information to the external PostgreSQL
```

Comment:
I think Pass should be capitalized here, and changed to something like "Provide"